nncf/model_creation.py

```diff
@@ -86,6 +87,16 @@ def create_compressed_model(model: Module, config: NNCFConfig,
     has special formats of data loader output or has additional *forward* arguments other than input tensors.
     Otherwise, the *forward* call of the model during graph tracing will be made with mock tensors according
     to the shape specified in the config object.
+    :param wrap_inputs_fn: will be used on the module's input arguments during a regular, non-dummy forward call before
```

Comment:
What's the point to use the future tense in the description of the function? New trend?

Reply:
Edited wording to justify the future tense usage